app/src/main/java/com/android/unio/ui/event/EventDetails.kt

```diff
@@ -159,7 +159,7 @@ fun EventScreen(
 fun EventScreenScaffold(
     navigationAction: NavigationAction,
     mapViewModel: MapViewModel,
-    event: Event,
+    event: Event?,
```

Comment:
You don't need the `?` here as the only call to EventScreenScaffold already ensures that the event is not null : `EventScreenScaffold(navigationAction, mapViewModel, event!!, organisers, eventViewModel, userViewModel)`

Reply:
Good spot thank you :rocket: